servers/graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/server/spring/GraphQLConfigurationProperties.kt

```diff
@@ -29,13 +30,21 @@ data class GraphQLConfigurationProperties(
     val packages: List<String>,
     /** Boolean flag indicating whether to print the schema after generator creates it */
     val printSchema: Boolean = false,
+    @NestedConfigurationProperty
```

Comment:
required extra metadata to generate GraalVM native image ([docs](https://docs.spring.io/spring-boot/docs/current/reference/html/native-image.html#native-image.advanced.nested-configuration-properties))